src/main/resources/config/liquibase/changelog/20170806142424_added_entity_Study.xml

```diff
@@ -45,6 +45,10 @@
             <column name="user_id" type="bigint">
                 <constraints nullable="true" />
             </column>
+            
+            <column name="apple" type="clob">
```

Comment:
Is this column name supposed to be 'apple'?

Reply:
Oopa, forget to dwlete old code

Reply:
nit: can you fix this column name?